style.css

```diff
@@ -56,14 +58,188 @@
     text-align: center;
   }
 
-  .social-media {
+  .main-text .social-media {
     display: flex;
     align-items: center;
     justify-content: center;
     list-style: none;
   }
 
-  .social-media img {
+  .main-text .social-media img {
     margin-right: 20px;
   }
+
+  .main-text .disable-button {
+    display: flex;
+    justify-content: center;
+    margin: 20px;
+  }
+
+  .my-work-text {
+    display: flex;
+    justify-content: center;
+    flex-direction: column;
+    margin: 30%;
+  }
+
+  .my-work-text h2 {
+    display: flex;
+    font-weight: 700;
+    font-size: 30px;
+    color: #172b4d;
+    text-align: center;
+    gap: 24px;
+  }
+
+  .my-work-text hr {
+    border: 2px solid #36b37e;
+    background-color: #36b37e;
+    width: 30px;
+    display: flex;
+    justify-content: center;
+    margin: 0% 50% 0% 50%;
+  }
+
+  .work-history {
+    display: grid;
+    grid-template-columns: 1fr;
+    grid-template-rows: 1fr 1fr 1fr 1fr 1fr 1fr;
+    margin: 144px 24px;
+    gap: 24px;
+  }
+
+  .about-me-text {
+    text-align: center;
+  }
+
+  .about-me-text h2 {
+    font-style: normal;
+    font-weight: 700;
+    font-size: 40px;
+    line-height: 52px;
+    align-items: center;
+    color: #172b4d;
+    flex: none;
+    order: 0;
+    flex-grow: 0;
+  }
+
+  .about-me-text p {
+    font-style: normal;
+    font-weight: 400;
+    font-size: 20px;
+    line-height: 32px;
+    align-items: center;
+    color: #42526e;
+    flex: none;
+    order: 1;
+    flex-grow: 0;
+  }
```

Comment:
- [ ] Same here as mentioned in the previous comment:point_up_2:

Reply:
resolved